paths/graphql/outline.md

```diff
@@ -0,0 +1,24 @@
+---
+
+---
+
+Top-level numbers individual pages?
+
+1. Introduction
+  - Motivation, present the problem that needs solving
+  - API as the solution (context: dynamic, consumable information)
+    - Examples of API usages (format: service, how API is used, link)
+1. First use (jump right in, no later than pg 2)
+  - instructions for navigating to GraphiQL on web UI
+  - interact with interface, show how it works, live data
+  - tell me why, can explain differences between rest and graphql
+1. Your first query (get personal statistic)
+  - short, direct explanation of a query
+  - instructions for building the first query
```

Comment:
Did the example of providing brief instructions on some of the items used in the first query to then go into further detail during the next course make sense? AKA, we used Hooks in Exercise 1, lets discuss what a Hook actually does, kind of thing...